--- src/lib/evas/common/evas_op_blend/op_blend_pixel_neon.c~	2017-04-12 09:30:19.000000000 +0100
+++ src/lib/evas/common/evas_op_blend/op_blend_pixel_neon.c	2017-05-26 17:54:43.955180353 +0100
@@ -318,7 +318,7 @@
 		// Add to 's'
 		"vqadd.u8	d4,	d8,d0		\n\t"
 
-		"vstr.32	d4,	[%[d]]		\n\t"
+		"vstr	d4,	[%[d]]		\n\t"
 		"add		%[d],   #8		\n\t"
 
 		"cmp 		%[tmp], %[d]		\n\t"
--- src/lib/evas/common/evas_op_copy/op_copy_color_neon.c~	2017-04-12 09:30:19.000000000 +0100
+++ src/lib/evas/common/evas_op_copy/op_copy_color_neon.c	2017-05-26 17:57:45.126405764 +0100
@@ -38,7 +38,7 @@
 		"blt		"AP"loopout		\n\t"
 
 	AP"dualloop:					\n\t"
-		"vstr.32	d0, [%[d]]		\n\t"
+		"vstr		d0, [%[d]]		\n\t"
 
 		"add		%[d], #8		\n\t"
 		"andS		%[tmp], %[d], $0x1f	\n\t"
