--- ./data/elementary/themes/edc/transitions.edc	2016-09-02 16:47:01.214802435 +0100
+++ ./data/elementary/themes/edc/transitions.edc~	2016-09-02 16:45:08.416360900 +0100
@@ -1,5 +1,5 @@
 group { name: "e/transpreview/0";
-   images.image: "img_example_1.png" LOSSY 92;
+   images.image: "img_example_1.png" LOSSY_ETC1 92;
    parts {
       part { name: "clip"; type: RECT;
          description { state: "default" 0.0;
@@ -16,7 +16,7 @@
 }
 
 group { name: "e/transpreview/1";
-   images.image: "img_example_2.png" LOSSY 92;
+   images.image: "img_example_2.png" LOSSY_ETC1 92;
    parts {
       part { name: "clip"; type: RECT;
          description { state: "default" 0.0;
