--- ./configure.ac~	2017-05-24 22:27:00.865768226 +0100
+++ ./configure.ac	2017-05-25 00:40:33.080355472 +0100
@@ -1988,7 +1988,9 @@
    [want_wayland="no"])
 
 if test "${want_wayland}" = "yes"; then
-   EFL_PKG_CHECK_STRICT([wayland-client >= 1.11.0 wayland-scanner >= 1.11.0 wayland-protocols >= 1.7])
+   EFL_PKG_CHECK_STRICT([wayland-client >= 1.11.0])
+   EFL_PKG_CHECK_STRICT([wayland-scanner >= 1.11.0]) 
+   EFL_PKG_CHECK_STRICT([wayland-protocols >= 1.7])
    PKG_CHECK_MODULES([WAYLAND], [wayland-scanner >= 1.11.0],
     [
       AC_ARG_VAR([wayland_scanner], [The wayland-scanner executable])
